### PR TITLE
aci_fibre_channel_policy: Refactor

### PR DESCRIPTION
**Pull request by dagwieers**
Please review

### CODE REVIEW

File: library/aci_fibre_channel_policy.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    state = module.params['state']
+
+    aci = ACIModule(module)
+
+    if fc_policy is not None:
+        # Work with a specific filter
+        path = 'api/mo/uni/infra/fcIfPol-%(fc_policy)s.json' % module.params
+    elif state == 'query':
+        # Query all filters
+        path = 'api/node/class/infra/fcIfPol.json'
```
Comment (jmcgill298) [inline library/aci_fibre_channel_policy.py]: The correct path i api/infra/class/fcIfPol.json

File: library/aci_fibre_channel_policy.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+'''
+
+from ansible.module_utils.aci import ACIModule, aci_argument_spec
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    argument_spec = aci_argument_spec
+    argument_spec.update(
+        fc_policy=dict(type='str', required=False),  # Not required for querying all policies
```
Comment (jmcgill298) [inline library/aci_fibre_channel_policy.py]: add alias "name"

**Comment by jmcgill298** [comment]
What do you all think about the name aci_fc_policy?

File: library/aci_fibre_channel_policy.py (hunk 2 above)
Comment (dagwieers) [inline library/aci_fibre_channel_policy.py]: Good one. So you think after a while you're not going to miss anything.
And then BOOM, you come around and shatter the dream :-D

File: library/aci_fibre_channel_policy.py (hunk 1 above)
Comment (dagwieers) [inline library/aci_fibre_channel_policy.py]: This was copied from the original module. Are you sure ?
(I guess the testing will catch this too...)

**Comment by dagwieers** [comment]
I very much would like to change the name.
@schunduri what do you think ?

**Pull request merged**